go_live.md

```diff
@@ -2,7 +2,6 @@
 
 ## DNS, SSL & SMTP
 
-* Go to https://www.cloudflare.com/ and add your site, if not already done
 * Set up your DNS records (e.g CNAMES to heroku, TXT records for SparkPost, ...)
 * If SparkPost has been set up with the renuoapp.ch domain and the project has its own domain now, set up SparkPost again with its own domain
 * Verify that SparkPost mails are working and the [sending domain is validated](https://app.sparkpost.com/account/sending-domains).
```

Comment:
mabe we should have a link here to the spark-post file. it's very important that we don't mess up here, otherwise domains get blocked.